src/components/DopplerPlus/UpgradeSuggestionForm.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { InjectAppServices } from '../../services/pure-di';
+import { Field } from 'formik';
+import {
+  EmailFieldItem,
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+const UpgradeSuggestionForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const fieldNames = {
+    email: 'email',
+    firstname: 'firstname',
+    lastname: 'lastname',
+    phone: 'phone',
+    range_time: 'range_time',
+    message: 'message',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
```

Comment:
Here you could use the upgrade @jcamposmk made. Here https://github.com/FromDoppler/doppler-webapp/blob/41f786d87c524e4bb9b80ec4b983f4e10efe6e7b/src/utils.ts#L317